data/dictionary.txt

```diff
@@ -1036,6 +1041,7 @@ constituant->constituent
 constituants->constituents
 constituion->constitution
 constituional->constitutional
+constructable->constructible
```

Comment:
http://www.collinsdictionary.com/dictionary/english/constructable

it seems to be acceptable, too.